frontend/playground-frontend/src/components/helper_functions/TalkWithBackend.js

```diff
@@ -57,8 +57,12 @@ const routeDict = {
 };
 
 async function train_and_output(choice, choiceDict) {
-  const trainResult = await sendToBackend(routeDict[choice], choiceDict);
-  return trainResult;
+  if (process.env.MODE === "dev") {
+    const trainResult = await sendToBackend(routeDict[choice], choiceDict);
+    return trainResult;
+  } else {
+    //TODO: submit request to sqs. return success or fail message!
```

Comment:
@farisdurrani this part is to be filled in. 

To test: follow the updated README.md instructions (keeping in mind to run the new npm command to populate dev environment variable). Everything should be working as normal

Reply:
Makes sense. But I think rn we need to have both prod and dev versions to be doing the same thing since we haven't implemented the SQS function

Reply:
I recommend for now, in both modes, have the same function running